spk/git/Makefile

```
@@ -10,7 +10,7 @@ DESCRIPTION = Git is a fast, scalable, distributed revision control system with
 DESCRIPTION_FRE = Git est un système de gestion de révision rapide, extensible et distribué avec un ensemble de commandes inhabituellement riche qui fournit à la fois des opérations de haut niveau et un accès complet aux structures de bas niveau.
 STARTABLE = no
 DISPLAY_NAME = Git
-CHANGELOG = "1. Update git to v2.37.0.<br/>2. Update OpenSSL to v1.1.1p.<br/>3. Update libcurl to v7.84.0."
+CHANGELOG = "1. Update git to v2.37.1 - Fix for CVE-2022-29187."
```

we updated openssl after git 2.37.0-23 was published.

```suggestion
CHANGELOG = "1. Update git to v2.37.1 - Fix for CVE-2022-29187.<br/>2. Update openssl to v1.1.1q"
```